resources/assets/enderio/config/SAGMillRecipes_Core.xml

```diff
@@ -2459,6 +2459,19 @@ addOreDictionaryTooltips=true
     </recipe>
   </recipeGroup>
 
+  <recipeGroup name="Natura">
+
+    <recipe name="Barley" energyCost="800">
+      <input>
+        <itemStack modID="Natura" itemName="barleyFood" />
+      </input>
+      <output>
+        <itemStack modID="EnderIO" itemName="itemPowderIngot" itemMeta="8" number="2" />
```

Comment:
I will merge it as is so its the same as wheat. I think it might be better balanced at 1:1.5, but food / bread production is not really much of a blanace issue either way as it is so easy to get and not that useful anyway regardless of the decision here.